source/game/system/GhostFile.cc

```diff
@@ -19,6 +19,9 @@ GhostFile::~GhostFile() = default;
 
 /// @addr{0x8051C530}
 void GhostFile::read(EGG::RamStream &stream) {
+    constexpr size_t RKG_MII_DATA_SIZE = 0x4A;
+    constexpr size_t RKG_USER_DATA_SIZE = 0x14;
```

Comment:
Is it worth adding this if unused right now?

Reply:
It's not unused - it skips bytes in the stream reading. I moved this from the header since it's specific to the reading.

Reply:
If only I had expanded the preview on Github...